### ISSUE TITLE
failed to install open-android-backup-companion.apk

### ISSUE THREAD

**Issue by Erz3** [mrrfv/open-android-backup#99]
## Device info
Manufacturer: BQ 
Android version: 8.1.0
Custom ROM/rooted: No
Model: Aquaris X

## System info
Available disk space: 29GB
Operating system: Ubuntu Mate running in a virtual machine with VirtualBox

---

**Describe the issue below.**

The script was backing up internal storage until I force a stop with "ctrl+c". Then I started again the script but now was precise download the APK because the aplication had deleted automatically. In this point, I stopped again the script's work and now I get the issue above.

In summary:

1- Start script
2- Force stop while backup
3- Start from the beginning
4- Force stop while download
5- Start from the beginning
6- Issue

> Open Android Backup will install a companion app on your device, which will allow for contacts and other data to be backed up and restored.
> The companion app is open-source, and you can see what it's doing under the hood on GitHub.
> Companion app already downloaded.
> Attempting to uninstall companion app.
> Installing companion app.
> Performing Streamed Install
> adb: failed to install open-android-backup-companion.apk: Failure [INSTALL_PARSE_FAILED_UNEXPECTED_EXCEPTION: Failed to parse /data/app/vmdl41105439.tmp/base.apk: AndroidManifest.xml]

I have tried restarting the smartphone but nothing change. I think the problem is the file mentioned in the output, maybe a corrupted file generated by force the stop while the download was in progress.

**Comment by mrrfv** [comment]
Could you please try redownloading the script again? It's possible that stopping it while it was doing something might've corrupted some files.
Additionally, check if you can install the app manually by going to https://github.com/mrrfv/open-android-backup/releases/tag/v1.0.15 on your device and installing `app-release.apk` - if that works, then the problem might lie within the fact that you're using VirtualBox.

**Comment by Erz3** [comment]
Ok, I will try tomorrow or in Monday. However I need stay clear something because I have explained bad the issue.

The sript has been backing up internal storage until I force a stop with "ctrl+c". Then I started again the script but now the APK needed to be downloaded because the aplication was deleted automatically. In this point, I stopped the script's work again and now I get the above issue.

I think the problem is the file mentioned in the output, maybe a corrupted file generated by force the stop while the download was in progress.
